src/lib/logical_query_plan/abstract_lqp_node.cpp

```diff
@@ -266,6 +269,12 @@ LQPColumnReference AbstractLQPNode::get_column(const QualifiedColumnName& qualif
   DebugAssert(colum_origin, "Couldn't resolve column origin of " + qualified_column_name.as_string());
   return *colum_origin;
 }
+//
+//LQPColumnReference AbstractLQPNode::get_column(const std::string& column_name) const {
+//  const auto iter = std::find(output_column_names().begin(), output_column_names().end(), column_name);
+//  DebugAssert(iter != output_column_names().end(), "Couldn't find column " + column_name);
+//  return LQPColumnReference{shared_from_this(), static_cast<ColumnID>(std::distance(output_column_names().begin(), iter))};
+//}
```

Comment:
?

Reply:
yeah, WIP, ignore the outcommented stuff ;)